src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1789,7 +1807,7 @@ class ContentExplorer extends Component<Props, State> {
                             isTouch={isTouch}
                             onCancel={this.closeModals}
                             item={selected}
-                            currentCollection={currentCollection}
+                            currentCollection={cloneDeep(currentCollection)}
```

Comment:
For some reason ContentPreview is modifying something inside of currentCollection directly, causing some issues on our side. Adding a deep clone fixes it. I assume that somewhere inside of ContentPreview, they used shallow instead of deep clone.